ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -315,8 +318,18 @@ class DrawerPresentationController: UIPresentationController {
         var contentFrame = bounds.inset(by: marginsForContentView())
 
         var contentSize = presentedViewController.preferredContentSize
+
+        let landscapeMode: Bool
+        if let windowSize = sourceViewController.view.window?.frame.size {
```

Comment:
will this work for iPad compact? Not sure if this window is the whole screen or just the narrow split view

Reply:
The window size is the narrow splitview.